vendor/k8s.io/kubernetes/pkg/kubectl/cmd/rollout/rollout_pause.go

```diff
@@ -119,17 +132,36 @@ func (o *PauseConfig) CompletePause(f *cmdutil.Factory, cmd *cobra.Command, out
 
 func (o PauseConfig) RunPause() error {
```

Comment:
I think copying is better here... I will have to make some generic option struct to make this work and I'm not sure it is worth (other than sharing this with `set`).